data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,277 @@ mission "Arachi Cartoonist 2"
+			`	Pairri draws your attention back to the line, where judges have the twenty teams line up. A chariot carrying the other two members of each team is attached to the Saryds, who seem to be working themselves up, hitting their hoofs repeatedly on the ground. Meanwhile, the Kimek and Arachi are securing large bags and pieces of fabric to their chariots.`
+			`	"In many ways we differ, when it comes to physical form," Pairri says. "So in the cooperative match, different tasks, each member of a team is assigned. For the sake of fairness, it is so."`
+			choice
+				`	"What are they carrying in those bags?"`
+				`	"So is pulling those chariots the 'task' of the Saryds?"`
+			`	A loud airhorn interrupts Pairri as she is about to answer, and the Saryds start galloping toward the wall. The Kimek and Arachi of each chariot do their best to stay on, as their Saryd teammates press on with little regard for the chariot's stability, focused on reaching the wall as fast as possible.`
+			`	It's a very tight race, but the Saryds begin breaking once they're getting closer, out of both exhaustion, and a need to avoid crashing into the wall. Once they arrive at the base, they immediately lay down to rest as the Arachi of each team unstrap the chariots, and the Kimek begin climbing the wall, carrying their small bags.`
+			`	Before long, they have all reached the top, opening their bags to grab a multitude of paddles, a pole that looks like a combination between a shoven and a pickaxe, and a long rope. "To cross the river, the primary goal of the game is," Pairri explains. "Prepare the raft for their team, the Kimek must."`
+			`	Back at the bottom, the Arachi are done helping their Saryd teammates out of the chariot straps, and prepare for the climb themselves, as they 'wear' the fabric pieces over their bodies, on which the Saryds lay, to be carried up by their teammates. You have your monitor focus on Pairri's friends again, and see that after ensuring Ulmodi is safely strapped on his back, Tumub begins his climb. Despite the stress of carrying someone much larger than him, his movements remain precise, never having more than one leg not touching the wall at a time, and carefully choosing the best spots on the wall to anchor himself on.`
+			`	Once they're at the top, Ulmodi gets up and removes the fabric from Tumub, now being his turn to carry his Arach teammate toward the now nearly complete raft. He lays the Tumub beside the raft and helps the Kimek properly secure the ropes. Once they're done, they help Tumub onto it, and start pushing the raft to the river. Ulmodi gets on it shortly before it touches the water, and the Kimek waits until the last second.`
```

Comment:
```suggestion
			`	Once Tumub reaches the top, Ulmodi gets dislodges himself and removes the harness from Tumub, before then lifting his Arach teammate into his arms so that he may climb onto the Saryd's back this time, as Ulmodie races to the by now nearly complete raft. Tumub climbs from Ulmodi's back into the raft and both their Kimek teammate finish properly securing the ropes. Once done, Ulmodi and the Kimek begin pushing the raft to the river, where Ulmodi now lifts up the Kimek and leaps into the raft just after it begins to float on the water.`
```